docs/2-guides/1-building-a-feedback-form/README.md

```diff
@@ -0,0 +1,354 @@
+
+<details>
+<summary><code>docker-compose.yml</code></summary>
+
+1. Give the `jahia` service a static IP address:
+
+   ```yaml
+   services:
+     jahia:
+       # ...
+       networks:
+         default:
+           ipv4_address: 172.16.1.100
+   ```
+
+2. Add the jCustomer containers:
+
+   ```yaml
+   services:
+     # ...
+
+     elasticsearch:
+       image: elasticsearch:7.17.28
+       ports:
+         - 9200:9200
+       environment:
+         discovery.type: single-node
+         cluster.name: jahia-es-cluster
+
+     kibana:
+       image: kibana:7.17.28
+       ports:
+         - 5601:5601
+       environment:
+         discovery.type: single-node
+         elasticsearch.hosts: http://elasticsearch:9200
+
+     jcustomer:
+       image: jahia/jcustomer:2.6
+       depends_on:
+         - elasticsearch
+       ports:
+         - 9443:9443
+         - 8181:8181
+         - 8102:8102
+       environment:
+         UNOMI_ELASTICSEARCH_ADDRESSES: elasticsearch:9200
+         UNOMI_ELASTICSEARCH_CLUSTERNAME: jahia-es-cluster
+         UNOMI_CLUSTER_PUBLIC_ADDRESS: http://localhost:8181
+         UNOMI_CLUSTER_INTERNAL_ADDRESS: https://jcustomer:9443
+         UNOMI_THIRDPARTY_PROVIDER1_IPADDRESSES: 172.16.1.100
+         UNOMI_THIRDPARTY_PROVIDER1_ALLOWEDEVENTS: login,updateProperties
+         UNOMI_ROOT_PASSWORD: karaf
+         UNOMI_HAZELCAST_TCPIP_MEMBERS: jcustomer
+   ```
+
+3. Add a network stack at the end of the file:
+
+   ```yaml
+   networks:
+     default:
+       ipam:
+         config:
+           - subnet: 172.16.1.0/24
+   ```
+
+</details>
+
+<details>
+<summary><code>docker/provisioning.yml</code></summary>
+
+Add this at the end of the file:
+
+```yaml
+# Install and start jExperience
+- installBundle:
+    - "mvn:org.jahia.modules/jexperience/3.6.2"
+    - "mvn:org.jahia.modules/jexperience-dashboards/1.0.0"
+  autoStart: true
+  uninstallPreviousVersion: true
+
+# Configure jExperience
+- editConfiguration: "org.jahia.modules.jexperience.settings"
+  configIdentifier: "global"
+  properties:
+    jexperience.jCustomerURL: "https://jcustomer:9443"
+    jexperience.jCustomerUsername: "karaf"
+    jexperience.jCustomerPassword: "karaf"
+    jexperience.jCustomerTrustAllCertificates: "true"
+    jexperience.jCustomerUsePublicAddressesForAdmin: "false"
+    jexperience.jCustomerKey: "670c26d1cc413346c3b2fd9ce65dab41"
+
+# Configure Kibana dashboards
+- editConfiguration: "org.jahia.modules.kibana_dashboards_provider"
+  properties:
+    kibana_dashboards_provider.kibanaURL: "http://kibana:5601"
+    kibana_dashboards_provider.kibanaUser: "elastic"
+    kibana_dashboards_provider.kibanaPassword: "ELASTIC_PASSWORD"
+    kibana_dashboards_provider.KibanaProxy.enable: "true"
+    kibana_dashboards_provider.KibanaProxy.cloud: "true"
+- installBundle:
+    - "mvn:org.jahia.modules/kibana-dashboards-provider/1.3.0"
```

Comment:
```suggestion
    - "mvn:org.jahia.modules/kibana-dashboards-provider/1.3.1"
```

https://github.com/Jahia/kibana-dashboards-provider/pull/153 should be released as 1.3.1